vtm/src/org/oscim/map/Animator2.java

```diff
@@ -41,13 +41,24 @@
     /**
      * The minimum changes that are pleasant for users.
      */
-    private static final float DEFAULT_MIN_VISIBLE_CHANGE_PIXELS = 0.5f;
+    private static final float DEFAULT_MIN_VISIBLE_CHANGE_PIXELS = 0.6f;
     private static final float DEFAULT_MIN_VISIBLE_CHANGE_RADIAN = 0.001f;
     private static final float DEFAULT_MIN_VISIBLE_CHANGE_SCALE = 1f;
 
-    private static final float FLING_FRICTION_MOVE = 0.9f;
-    private static final float FLING_FRICTION_ROTATE = 1.0f;
-    private static final float FLING_FRICTION_SCALE = 1.2f;
+    /**
+     * The friction scalar for fling movements (1 as default).
+     */
+    public static float FLING_FRICTION_MOVE = 1.0f;
+
+    /**
+     * The friction scalar for fling rotations (1 as default).
+     */
+    public static float FLING_FRICTION_ROTATE = 1.2f;
+
+    /**
+     * The friction scalar for fling scales (1 as default).
+     */
+    public static float FLING_FRICTION_SCALE = 1.2f;
```

Comment:
In docs mean `(1.2 as default)` in rotate and scale cases?

Reply:
Yes, only wanted to give a reference to see whats the main scalar.

Reply:
"as default" means what library has if users don't change it, in rotate/scale case is `1.2`.
If we need to mention `1` value, then should express it differently, like "as base"?

Reply:
Yes, that sounds better! Sorry for that...